src/Module/Special/HTTPException.php

```diff
@@ -101,8 +99,9 @@ public function rawContent(\Friendica\Network\HTTPException $e)
 
 		// We can't use a constructor parameter for this response object because we
 		// are in an Exception context where we don't want an existing Response.
+		$reason = ($e instanceof NetworkHTTPException) ? $e->getDescription() : $e->getMessage();
```

Comment:
Won't `$e` always be an instance of `NetworkHTTPException` because of the function parameter type-hinting?

Reply:
`$e` can also be an `\Exception` catched in line 92/90.

Reply:
Good catch, `$e`  in the inside catch block is never used and should be renamed (we can't simplify to `} catch (Throwable) {` just yet because of PHP 7.4 support).

Reply:
Ok, fixed.